changelogs/fragments/651-fix-postgresql_set-warning.yaml

```diff
@@ -0,0 +1,4 @@
+---
+bugfixes:
+  - postgresql_set - only display a warning about restarts, when
+    restarting is needed
```

Comment:
```suggestion
    restarting is needed (https://github.com/ansible-collections/community.general/pull/651).
```

Reply:
when we don't have a related issue we put a reference to a pr itself. Maybe there is not in the doc now, but the relevant changes were merged recently.

Reply:
@Andersson007 I put the PR into the filename. But sure!